doc/api/events.md

```diff
@@ -581,6 +581,26 @@ being removed. This will not impact the order in which listeners are called,
 but it means that any copies of the listener array as returned by
 the `emitter.listeners()` method will need to be recreated.
 
+When a single function has been added as a handler multiple times for a single
+event (as in the example below), `removeListener()` will remove the most
+recently added instance. In the example, for instance, the `once('ping')`
```

Comment:
Maybe we can reword this to not use `instance` twice in a row in different meanings?

Reply:
```suggestion
recently added instance. In the example the `once('ping')`
```